### DISCUSSION TITLE
Multi Slice view

### DISCUSSION THREAD

**Discussion by skpathak** [Kitware/trame#238] [Trame/VTK] [answered]
Hello All, 

I am trying to visualize the Multislice view of a vti data with 256,256,128 voxels using trame. I can create the visualization using vtk.js using vtkImageSlice, and vtkImageMapper using the set(I, J, K)Slice method. But I can't replicate the same using trame. I have used vtkImageSliceMapper, but it is giving an error in the js console that the server does not allow the method (405 error). 

I don't know if I am missing anything. I would appreciate any suggestions or help you can give.

Thanks,
SUdhir

**Comment by jourdain** [comment, marked as answer]
Trame does not expose vtkImageMapper from the vtk.js side of things. 
If you really want to use trame with such setup, that will require some coding to either expose vtkImageMapper or bundle it into a custom widget. 

HTH

**Comment by guozhi72** [comment]
shouldn't vtkImageMapper be called in vtk python side? Do you mean using trame, currently trame doesn't surpport render 3d Volume slice by slice?

**Reply by jourdain** [reply]
You can use vtk.js (in trame) without VTK on the server side. But only a subset got exposed because of the inherent limitation of such setup.
